tests/smoke-bundler-group-rules.yaml

```
@@ -0,0 +1,465 @@
+input:
+    job:
+        package-manager: bundler
+        allowed-updates:
+            - dependency-type: direct
+              update-type: all
+        dependency-groups:
+            - name: ruleset
+              rules:
+                patterns:
+                    - rack
+                    - rubocop
+        experiments:
+            grouped-updates-prototype: true
+        ignore-conditions:
+            - dependency-name: rubocop
+              source: tests/smoke-bundler-grouped.yaml
+              version-requirement: < 1.48.1
+        source:
+            provider: github
+            repo: dependabot/smoke-tests
+            directory: /bundler
+            commit: f558406ea0f067dcb5e223e257dcd91c27ea3a85
+            hostname: github.com
+            api-endpoint: https://api.github.com/
+        commit-message-options: {}
+        credentials-metadata:
+            - host: github.com
+              type: git_source
+            - host: github.com
+              type: git_source
```

You probably only need one of these, although I doubt it makes any difference. 

```suggestion
            - host: github.com
              type: git_source
```